src/output/icons.rs

```diff
@@ -58,7 +58,7 @@ impl Icons {
     const LANG_KOTLIN: char     = '\u{e634}';  // 
     const LANG_OCAML: char      = '\u{e67a}';  // 
     const LANG_PERL: char       = '\u{e67e}';  // 
-    const LANG_PHP: char        = '\u{e73d}';  // 
+    const LANG_PHP: char        = '\u{e608}';  // 
```

Comment:
This one needs to be discussed

Reply:
I think the elephant is more visually appealing than a logo with tiny letters that just says "PHP". Same thing with the assembly language icon - it doesn't have to be the pound sign, but it should be better than a glyph that just says "ASM".